### ISSUE TITLE
Go to home location button is overlapping move button/icon

### ISSUE THREAD

**Issue by danieldegroot2** [pietervdvn/MapComplete#1787]
**Expected behaviour**
Two different icons/alt texts belong to two buttons, so there should be two buttons here, one for movement and one for home location.

**Current behaviour**
The icon for home location mostly hides the movement button, only an up arrow sticks out above it. There is just enough of it shown to display the alt text.

This happens regardless of browser zoom level.
(For me personally, above 150% zoom, the movement icon seems to move to the right, meaning it is possibly not centered on the column, but aligned to the right)

**Screenshot(s)**
Overlapping icons. Go to home location looks like a compass needle..
<img src="https://github.com/pietervdvn/MapComplete/assets/67521919/4c3f4828-021b-4c71-ac2f-df7f48af48cf" width="400"/>

There is another button here, but it is hidden.
<img src="https://github.com/pietervdvn/MapComplete/assets/67521919/d1734b73-df50-41d2-bd64-e6e4bf9b6927" width="400"/>

---

(_Disclaimer: I personally have a high dpi screen._)

**Comment by pietervdvn** [comment]
> Edit: I suppose it also happens to appear anywhere on screen when not zoomed in properly and after moving around cursor at random.

This is a standard tooltip, it appears automatically when the cursor doesn't move for a few seconds.

> Would it be a good idea to have a seperate compass button where the user can re-align the map to north? 

No, there is too little room for this. In 'settings', there is the option to disable rotation though.
